jmetal-core/src/main/java/org/uma/jmetal/solution/impl/DefaultDoubleSolution.java

```diff
@@ -77,4 +74,11 @@ public DefaultDoubleSolution copy() {
   public String getVariableValueString(int index) {
     return getVariableValue(index).toString() ;
   }
+  
+  private void initializeDoubleVariables() {
+    for (int i = 0 ; i < problem.getNumberOfVariables(); i++) {
+      Double value = randomGenerator.nextDouble(getLowerBound(i), getUpperBound(i)) ;
```

Comment:
No, it's OK, I did not notice the actual difference is the function, not the formula.

nextDouble() without arguments returns a value in [0;1], while nextDouble(min, max) returns a value between min and max, so the result is the same at the end. Well actually I did not check but this is the usual semantic, so I guess it is the same here.